testsuite/testsuite-microprofile-jwt/src/test/java/org/wildfly/swarm/microprofile/jwtauth/keys/ClasspathResourceTest.java

```diff
@@ -43,18 +39,12 @@
 public class ClasspathResourceTest {
 
     @Deployment
-    public static Archive createDeployment() throws Exception {
-
-        String fileName = "./src/test/resources/keys/public-key.pem";
-        pubKeyFile = new File(fileName);
-
-        WARArchive deployment = ShrinkWrap.create(WARArchive.class, "pubkey.war");
-        deployment.addClass(ResourceLoadingServlet.class);
-        deployment.addAsResource(pubKeyFile, "/public-key.pem");
+    public static Archive<?> createDeployment() throws Exception {
 
-        /*URL url = Thread.currentThread().getContextClassLoader().getResource("cp-resource.yaml");
-        assertThat(url).isNotNull();
-        File projectDefaults = new File(url.toURI());*/
+        JAXRSArchive deployment = ShrinkWrap.create(JAXRSArchive.class, "pubkey.war");
+        deployment.addResource(TestApplication.class);
+        deployment.addResource(KeyLoadingService.class);
```

Comment:
Np. I thought JAXRSArchive was somehow helping in glueing JAX-RS resources :-)

Reply:
Personally, when I use `@Deployment` instead of `@DefaultDeployment`, I default to the standard ShrinkWrap `JavaArchive` or `WebArchive`, and only resort to the WildFly Swarm archive classes if I need to do something specific.